BraintreeCore/src/main/java/com/braintreepayments/api/PostalAddress.kt

```diff
@@ -1,78 +1,36 @@
 package com.braintreepayments.api
 
-import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
 import android.text.TextUtils
-import android.os.Parcel
+import android.os.Parcelable
 
-// NEXT_MAYOR_VERSION: transform to data class and check for @Parcelize annotation
 /**
  * Java object representing a postal address
  */
-open class PostalAddress : Parcelable {
-    open var recipientName: String? = null
-    open var phoneNumber: String? = null
-    open var streetAddress: String? = null
-    open var extendedAddress: String? = null
-    open var locality: String? = null
-    open var region: String? = null
-    open var postalCode: String? = null
-    open var sortingCode: String? = null
-    open var countryCodeAlpha2: String? = null
-
-    constructor()
+@Parcelize
+data class PostalAddress(
+    var recipientName: String?,
```

Comment:
Do you have a preference on making all data class members to be `val`s?
The data class object would then be immutable and copy() constructor could be used to make changes.

Reply:
Hmm in general if it can be a `val` it probably should be... but in this case these parameters should actually be mutable I think. It also lets merchants use `apply` when constructing which seems to be a common pattern. But I could go either way on this one. @sshropshire do you have a preference?

Reply:
Yeah it's totally for convenience to allow merchants to use `apply`. When we were 100% Java, we used to offer a builder pattern to give a similar Developer Experience to merchants. I personally like the flexibility of it, but I also like immutable types so I'm 50/50.

Reply:
Last thought: Immutability may be a heavy concept for a dev that's just learning.

Reply:
Ok yeah for the `apply` convenience I'm going to leave these as `var`